Add reference docs for The Pop-Up API, and my own author details (#4069)
GoogleChrome/developer.chrome.com: site/en/docs/web-platform/popup-api/backdrop-pseudo-element/index.md

---
layout: 'layouts/doc-post.njk'
title: ::backdrop
description: Allows styling of content behind the pop-up while the pop-up is open.
subhead: Allows styling of content behind the pop-up while the pop-up is open.
authors:
  - chrisdavidmills
date: 2022-10-19
updated: 2022-10-19
---

> The **`::backdrop`** [CSS](https://developer.mozilla.org/docs/Web/CSS) [pseudo-element](https://developer.mozilla.org/docs/Web/CSS/Pseudo-elements) sits behind an open [pop-up](/docs/web-platform/popup-api/) in the stacking order, but in front of the rest of the document, and spans the entire width and height of the viewport. It allows the rest of the content to be styled while the pop-up is open — for example you might want to blur or darken it.

Reviewer: ```suggestion
The **`::backdrop`** [CSS](https://developer.mozilla.org/docs/Web/CSS) [pseudo-element](https://developer.mozilla.org/docs/Web/CSS/Pseudo-elements) sits behind an open [pop-up](/docs/web-platform/popup-api/) in the stacking order, but in front of the rest of the document, and spans the entire width and height of the viewport. It allows the rest of the content to be styled while the pop-up is open—for example you might want to blur or darken it.
```

`::backdrop` neither inherits from nor is inherited by any other elements.

## Syntax

```css
::backdrop
```

## Example

> Here we are darkening the rest of the content while the pop-up is shown:

Reviewer: ```suggestion
In the following example the rest of the content is darkened while the pop-up is shown:
```
